generators/app/index.js

```diff
@@ -0,0 +1,502 @@
+"use strict";
+var chalk = require("chalk");
+var pkg = require("../../package.json");
+var _ = require("lodash");
+var proptypes = require("prop-types");
+var mkdirp = require("mkdirp");
+
+// var reduxBeacon = require("redux-beacon"); TODO OBLIGATORIO
+
+var Generator = require("yeoman-generator");
+
+class GeneratorReact extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  welcome() {
+    var kickoffWelcome =
+      chalk.blue("\n  ██╗  ██╗██╗ ██████╗██╗  ██╗ ") +
+      chalk.white("██████╗ ███████╗███████╗") +
+      chalk.blue("\n  ██║ ██╔╝██║██╔════╝██║ ██╔╝") +
+      chalk.white("██╔═══██╗██╔════╝██╔════╝") +
+      chalk.blue("\n  █████╔╝ ██║██║     █████╔╝ ") +
+      chalk.white("██║   ██║█████╗  █████╗") +
+      chalk.blue("\n  ██╔═██╗ ██║██║     ██╔═██╗ ") +
+      chalk.white("██║   ██║██╔══╝  ██╔══╝") +
+      chalk.blue("\n  ██║  ██╗██║╚██████╗██║  ██╗") +
+      chalk.white("╚██████╔╝██║     ██║") +
+      chalk.blue("\n  ╚═╝  ╚═╝╚═╝ ╚═════╝╚═╝  ╚═╝ ") +
+      chalk.white("╚═════╝ ╚═╝     ╚═╝") +
+      "\n\n  " +
+      chalk.blue.bold("Welcome Woloxer to the React kickoff") +
+      "\n\n  ";
+
+    this.log(kickoffWelcome);
+  }
+
+  askFor() {
+    var prompts = [
+      {
+        type: "input",
+        name: "projectName",
+        message: "Your Project name",
+        default: "Your project name",
+        store: true,
+        validate: val =>
+          String(val).match(/^[$A-Z_][0-9A-Z_$]*$/i)
+            ? true
+            : `${val} is not a valid name for a project. Please use a valid identifier name (alphanumeric).`
+      },
+      {
+        type: "input",
+        name: "projectDescription",
+        message: "Description",
+        default: "Project description",
+        store: true,
+        validate: val =>
+          String(val).match(/^[$A-Z_][0-9A-Z_$]*$/i)
+            ? true
+            : `${val} is not a valid description for a project. Please use a valid description (alphanumeric).`
+      },
+      {
+        type: "input",
+        name: "repoUrl",
+        message: "What is the git repo url for this project?",
+        store: true
+      },
+      {
+        type: "confirm",
```

Comment:
I'd remove all these confirms and decide which libraries we want to always include and which libraries we don't.

When you kickoff a project you probably don't know if you will need recharts, carousel, mobileDetect, etc. Forcing a decision over something you don't know is more a problem than a solution.